.changeset/@envelop_immediate-introspection-2235-dependencies.md

```diff
@@ -0,0 +1,8 @@
+---
+'@envelop/immediate-introspection': patch
```

Comment:
```suggestion
'@envelop/immediate-introspection': major
```

Reply:
done!

Reply:
hi @enisdenjo, let me know if there is anythig else that I can do at this point.